src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age !== undefined && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried === false ? (
+          `I am not married`
+        ) : (
+          `${partnerName} is my ${sex === 'm' ? `wife` : `husband`}`
+        )}
```

Comment:
```suggestion
        {isMarried ? (
          `${partnerName} is my ${sex === 'm' ? `wife` : `husband`}`
        ) : (
          `I am not married`
        )}
```
isMarried already boolean. in 99,99% cases no need to compare boolean value with `===`